build.gradle

```diff
@@ -223,6 +223,17 @@ task pushDockerImage {
     }
 }
 
+task testInitNakadi(type: Exec) {
```

Comment:
Cool! I'd better use
environment "SPRING_PROFILES_ACTIVE", "acceptanceTest"
because intention is more clear.